frontend/src/pages/Register/Register.jsx

```diff
@@ -14,29 +14,32 @@ import TextLink from '../../components/TextLink';
 const Register = () => {
 
   const params = useParams();
-  let paramEmail = params.email; 
+  let paramEmail = params.email;
   /* eslint-disable no-unused-vars */
-  let paramEmailVerificationToken = params.emailVerificationToken; 
+  let paramEmailVerificationToken = params.emailVerificationToken;
   /* eslint-disable no-unused-vars */
-  let paramFamily = params.family; 
+  let paramFamily = params.family;
 
```

Comment:
You can try to use destructuring to declare variables that you get from `params`
`const { paramEmail, paramEmailVerificationToken, paramFamily  } = useParams();`